lib/dependencies.js

```diff
@@ -44,6 +44,8 @@ let versionMap = {
   "karma-babel-preprocessor": "^6.0.1",
   "karma-typescript-preprocessor": "^0.2.1",
   "minimatch": "^3.0.2",
+  "requirejs": "^2.3.2",
+  "requirejs-text": "^2.0.12",
```

Comment:
this is wrong. version 2.0.12 doesn't work with aurelia, I've tried. Its also not maintained, nor official. `requirejs` repo has the plug-in inside it and its the only place its up to date.